guardian/typerighter
# Consolidate application code in a single folder

## What does this change?

This PR consolidates application code in a single folder. 

In this way, it's clear to devs as they look at the project where to go for app code (global or per-project), and where to go for infra, configuration etc.

Done as I realised that adding whilst adding a 'common' folder it wasn't clear it was going to be code related.

## How to test

This is a no-op – the app should run locally, test, and deploy as it did before.

This PR has many changes, but they should consist of either path changes and/or file moves.

## How can we measure success?

Arguably slightly less confusion when navigating repo? 😀
